mono/metadata/threads.c

```diff
@@ -4836,8 +4836,9 @@ MonoException*
 mono_thread_get_undeniable_exception (void)
 {
 	MonoInternalThread *thread = mono_thread_internal_current ();
+	MonoException *exc = mono_atomic_cas_ptr (&thread->abort_exc, NULL, NULL);
```

Comment:
Should this be casted?

Reply:
I don't think it's needed in C code, like in C++.

Reply:
Right yeah. I just saw that it was casted elsewhere when `mono_atomic_cas_ptr` was used. If we ever turn on werror I imagine this will pop up.

Reply:
I pushed change with casting.